products/storage/metadata/storage_move_file.toml

```
@@ -62,12 +62,12 @@ terraform = ["tf/storage_move_file/single_bucket_move.tf"]
 command = "gsutil ls gs://$BUCKET_NAME/ | grep $NEW_FILE_NAME"
 
 [tests.cleanup]
-# Make sure the template is deleted. If the command can fail, use `|| true` to supress the failure.
-# In this case, it WILL fail, if the test was successful, so we want to ignore this failure.
+# Make sure the file is deleted. If the command can fail, use `|| true` to supress the failure.
 # Since Terraform was used to set up the environment, it will be automatically used to clean up using
-# `terraform destroy`. However, since some resources might not longer be under Terraform control, we have an option to
+# `terraform destroy`. However, since some resources might no longer be under Terraform control, we have an option to
 # execute some commands before `terraform destroy` takes place.
-commands = ["gsutil rm -f gs://$BUCKET_NAME/$NEW_FILE_NAME"]
+
+commands = ["gsutil rm -f gs://$BUCKET_NAME/$NEW_FILE_NAME || true"]
```

This makes sense for prototype, it might be good to move the error suppression into a separate setting, maybe a "warnOnFailure" that would allow the tool to suppress the error but still log as part of the tool's UX that something went wrong.

---

This could work when we have only one command, but if there are multiple commands and we want to ignore error for only some of them, it gets more complicated. 

Definitely something to consider if this gets to evolve to full project.